tensorboard/plugins/text_v2/text_demo_v2.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from absl import app
+from absl import logging
+import tensorflow as tf
+from tensorboard.plugins.text.summary_v2 import text
+
+tf.compat.v1.enable_v2_behavior()
+
+# Directory into which to write tensorboard data.
+LOGDIR = "/tmp/text_demo"
+
+# Number of steps for which to write data.
+STEPS = 16
+
+
+def simple_example(step):
+    # Text summaries log arbitrary text. This can be encoded with ASCII or
+    # UTF-8. Here's a simple example, wherein we greet the user on each
+    # step:
+    step_string = tf.as_string(step)
+    greeting = tf.strings.join(["Hello from step ", step_string, "!"])
```

Comment:
I think `tf.constant('hello from step %d!' % step)` might be simpler to write and read. Here and below. Also, feel free to put a jovial emoji to showcase the unicode :)

Reply:
done